test/qpf.lean

```diff
@@ -143,8 +143,7 @@ begin
   existsi [bool,ff,ff,λ a : unit, tt,λ a : unit, ff], split,
   { apply quot.sound, simp [foo.R,qpf.abs,qpf._match_1], },
   { simp! only [set.range,set.ext_iff], simp,
```

Comment:
Could you replace this `simp` with the output of `squeeze_simp` so we can avoid more breakage?